examples/with-eth-passkeys-galore/src/pages/index.tsx

```diff
@@ -178,6 +278,94 @@ export default function Home() {
     }
   };
 
+  const signTransactionWithAAClient = async (
+    data: signTransactionFormData,
+    getSmartAccount: (
+      data: signTransactionFormData
+    ) => Promise<BiconomySmartAccountV2>,
+    options?: { nonceOptions?: { nonceKey: number } }
+  ) => {
+    const { destinationAddress, amount } = data;
```

Comment:
tiny nit: do we want to do any input validation?

Reply:
we should in a more serious application but will leave it out of this PR for now :)